packages/e2e/src/page-objects/messages.page.ts

```diff
@@ -129,6 +129,34 @@ class MessagesPage extends Page {
   > {
     return this.threadDropdownMarkAsReadButton.$("p")
   }
+  /** Returns list of last message text displayed on the conversation list and true/false depending on message unread/read status*/
+  async getLastMessages() {
+    const allLastMessages = await $$('//*[@data-testid="thread-last-message"]')
+
+    const messagesContent: string[] = await browser.executeAsync((done) => {
```

Comment:
`messagesContent` or `messagesContents`? 🤔 This is a list of the `messagesContent`?

`pseudoElements` it the same as `messagesContent`? Maybe it should be the same variable name to avoid to many new variable name contexts.

Reply:
cff6d8e82fa21faa9d1e36635830277c443a0981